templates/docs/base_docs.html

```diff
@@ -0,0 +1,312 @@
+    {% for element in nav_items %}
+    <li class="p-side-navigation__item">
+      {% if element.navlink_href %}
+      <a
+        class="p-side-navigation__link {% if expandable and element.children %}is-expandable{% endif %}"
+        href="{{ element.navlink_href }}"
+        {% if expandable and element.children %}aria-expanded={% if expanded or element.is_active %}"true"{% else %}"false"{% endif %}{% endif %}
+        {% if element.is_active and not element.navlink_fragment %}aria-current="page"{% endif %}
+      >{{ element.navlink_text }}</a>
+      {% else %}
+        <strong
+          class="p-side-navigation__text {% if expandable and element.children %}is-expandable{% endif %}"
+          {% if expandable and element.children %}aria-expanded={% if expanded %}"true"{% else %}"false"{% endif %}{% endif %}
+          {% if element.is_active %}aria-current="page"{% endif %}
+        >{{ element.navlink_text }}</strong>
+      {% endif %}
+
+      {% if expandable %}
+        {% if element.children %}
+            <button class="p-side-navigation__expand" aria-expanded={% if element.is_active or element.has_active_child %}"true"{% else %}"false"{% endif %} aria-label="show submenu for {{ element.navlink_text }}"></button>
+        {% endif %}
+        {{ create_navigation(element.children, expandable, element.is_active or element.has_active_child) }}
+      {% else %}
+        {% if element.children %}
+          {{ create_navigation(element.children, expandable) }}
+        {% endif %}
+      {% endif %}
+    </li>
+    {% endfor %}
+  </ul>
+{% endmacro %}
+
+<div class="l-docs">
+  <div class="l-docs__header">
+    
+    {% include "includes/header.html" %}
+
+    <section id="search-docs" class="p-strip--light is-shallow is-bordered l-docs__subgrid">
+      <div class="l-docs__main">
+        <div class="row">
+          <form class="p-search-box u-no-margin--bottom" action="/docs/search">
+            <input type="search" class="p-search-box__input" name="q" {% if query %}value="{{ query }}"{% endif %} placeholder="Search documentation" required/>
+            <button type="reset" class="p-search-box__reset" alt="reset"><i class="p-icon--close">Close</i></button>
+            <button type="submit" class="p-search-box__button" alt="search"><i class="p-icon--search">Search</i></button>
+          </form>
+        </div>
+      </div>
+    </section>
+  </div>
+
+  <div class="l-docs__sidebar">
+    <div class="l-docs__sticky-container">
+      {% if versions | length > 1 %}
+        <label for="version-select" class="u-hide">Version</label>
+        <select name="version-select" id="version-select" onChange="window.location.href=this.value">
+        {% for version in versions %}
+          {% set active = docs_version == version['path'] %}
+          <option value="{{ version_paths[version['path']] }}"{% if active %} selected{% endif %}>{{ version['version'] }}</option>
+        {% endfor %}
+        <select>
```

Comment:
Looks like an incorrect closing tag?
```suggestion
        </select>
```